src/seedu/addressbook/commands/attendance/UpdateAttendanceCommand.java

```diff
@@ -19,7 +19,7 @@
     public static final String COMMAND_WORD = "attendance";
     public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n"
             + "Updates the attendance of a student. \n"
-            + "Parameters: indexOfPerson d/dd-mm-yyyy att/attendance \n"
+            + "Parameters: INDEX d/dd-MM-yyyy att/ISPRESENT \n"
             + "\tExample: " + COMMAND_WORD + " " + "1 d/29-09-2018 att/1 \n"
```

Comment:
Intriguing, why is only the month capitalized lawl?

Reply:
I think it should be DATE instead of the format

Reply:
Mick, u literally told me to change it from date to the format.

Tek In, mm means minutes. MM is month

Reply:
Its just odd that the rest uppercase everything, while yours is only the month

Reply:
i'll change all to caps then

Reply:
Sorry about that, I was comparing with your other commands. However, now that I look through everyone's commands, for PARAMETERS, they are words and not format related